src/components/DataView/DataViewTHead.tsx

```diff
@@ -61,7 +64,8 @@ const StyledTh = styled.th`
 
 	& > .columnHeader > .icon {
 		visibility: hidden;
-		font-size: 20.58px;
+		height: 21.34px;

```

Comment:
I'm not sure where these sizes come from, the figma has 24 x 24px
![image](https://user-images.githubusercontent.com/23409284/203439037-a10f5d94-8563-4921-9cda-634ba94e9d65.png)